### PR TITLE
Reading focus mode.

### PR DESCRIPTION
**Pull request by dbrant**
https://phabricator.wikimedia.org/T254771

### CODE REVIEW

File: app/src/main/java/org/wikipedia/views/ObservableWebView.kt [hunk 1]
```diff
@@ -135,8 +135,10 @@ class ObservableWebView : WebView {
             MotionEvent.ACTION_UP -> {
                 if (abs(event.x - touchStartX) <= touchSlop &&
                         abs(event.y - touchStartY) <= touchSlop) {
-                    if (onClickListeners.any { it.onClick(event.x, event.y) }) {
-                        return true
+                    if (hitTestResult.type == HitTestResult.UNKNOWN_TYPE) {
```
Comment (cooltey) [inline app/src/main/java/org/wikipedia/views/ObservableWebView.kt:138]: Could you please add a note here to explain why we should check here and why `HitTestResult.UNKNOWN_TYPE`?

**Review by cooltey — COMMENTED** [review]
Looks good to me and only one comment below:

File: app/src/main/java/org/wikipedia/views/ObservableWebView.kt (hunk 1 above)
Comment (dbrant) [reply app/src/main/java/org/wikipedia/views/ObservableWebView.kt:141]: done!

**Review by cooltey — APPROVED** [review]
Looks good to me now. Just curious that is it possible to send the reading focus mode variable to the `mobile-html` in the initial setup? So that we will not see the edit buttons pop up and disappear every time when we jump to another article.

**Comment by dbrant** [comment]
> Looks good to me now. Just curious that is it possible to send the reading focus mode variable to the `mobile-html` in the initial setup? So that we will not see the edit buttons pop up and disappear every time when we jump to another article.

@cooltey Good point, but I don't think the page-library currently supports passing an "editable" variable in the initial setup. I agree this would be useful, but we'll need to add it to PCS, and then follow it up with another client-side patch.

**Pull request merged**